Dockerfile

```diff
@@ -21,13 +21,24 @@ ADD tests /docker/tests
 ADD music /docker/music
 ADD blender /docker/blender
 ADD nginx /docker/nginx
+ADD config /docker/config
 
 ## Install relevant packages
 
-RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get -y upgrade && apt-get install -y sudo lsb-release
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get -y upgrade && apt-get install -y sudo \
```

Comment:
Personally, I'd go with 
```
...install \
    sudo \
   ...
```
as it keeps the indent standard, and _every_ dep is consistently on it's own line. Minor, and if it's merged without that, no biggie.

Reply:
Also - I would recommend [squashing](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git) these commits.

This is a nice, very discrete PR, with only one aspect of change (as preferred!), and no need to keep PR corrections in the project changelog.

Reply:
My first time to use squashing, seems like it is working.